service/history/transferQueueActiveProcessor.go

```diff
@@ -867,10 +878,16 @@ func (t *transferQueueActiveProcessorImpl) processRecordWorkflowStarted(task *pe
 	startTimestamp := executionInfo.StartTimestamp.UnixNano()
 	executionTimestamp := getWorkflowExecutionTimestamp(msBuilder).UnixNano()
 
+	var visibilityMemo map[string][]byte
```

Comment:
There is space to optimize, since above getWorkflowExecutionTimestamp also get start event